ceph-dev-new-trigger/config/definitions/ceph-dev-new-trigger.yml

```diff
@@ -128,6 +128,38 @@
                     FORCE=True
                     DISTROS=centos8
                     FLAVOR=notcmalloc
+      # We need only centos8 for crimson ceph container for now
```

Comment:
This isn't necessary.  WIP branches are detected in this last conditional step: https://github.com/ceph/ceph-build/blob/master/ceph-dev-new-trigger/config/definitions/ceph-dev-new-trigger.yml#L131-L162

You only need to add `centos8` after `focal` here: https://github.com/ceph/ceph-build/blob/master/ceph-dev-new-trigger/config/definitions/ceph-dev-new-trigger.yml#L161

Reply:
ack!